attic/README.md

```diff
@@ -0,0 +1,19 @@
+# python-gluster-mgmt-client
```

Comment:
Why did you create this directory? If you really want a named way to get to the source code as it was before the code generation approach, I strongly recommend creating a branch rather than renaming a bunch of stuff into a directory.